### PR TITLE
Shellshock rework

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

The Hand-held version of the Shellshock loses it's ability to fire beanbags, and gains a mode that can fire a weaker version of the projectile frag grenades the china lake uses. It usually deals between 35 and 50 damage on a direct hit to an unarmored human, but rarely it can deal as much or as few damage as 20 or 60 due to chance-based same-turf hits.

The hand-held version also gains the ability to shoot once more if its cell is out of charge, which consumes the cell. The charge cost for the grenade launcher is so high that it will always consume the cell. Additionally, the hand-held shellshock must now be wielded, to increase the time to reload

This configuration encourages using one of the shotgun firemodes until the cell is empty, and then using up the cell with a grenade or another slug/buckshot. if you want to, you can also use the grenade mode immediately, but you'll get nowhere near as much mileage out of your cells this way. Or you can use high-quality cells instead for a high amount of shots, but need to be careful not to accidentally burn out your precious cell.

The Blitzshell version remains unaltered.

## Why It's Good For The Game

It's cooler than trash

## Changelog
:cl:
tweak: reworked the shellshock to be able to shoot grenades, as well as being able to trade an emty cell for another shot
/:cl:

### CODE REVIEW

**Review by genessee596 — CHANGES_REQUESTED** [review]
Add a 'safe' mode that will prevent burning out the cell (and disallow grenade firing) for people who don't want to accidentally lose their atomcell or something

File: code/modules/projectiles/guns/energy/shrapnel.dm [hunk 1]
```diff
@@ -11,33 +11,49 @@
...
+	else
+		return new projectile_type(src)
 
+/obj/item/weapon/gun/energy/shrapnel/attackby(obj/item/I, mob/user)
+	..()
+	if(I.has_quality(QUALITY_BOLT_TURNING))
+		if(I.use_tool(user, src, WORKTIME_NEAR_INSTANT, QUALITY_BOLT_TURNING, FAILCHANCE_VERY_EASY, required_stat = STAT_MEC))
+			if(consume_cell)
+				consume_cell = FALSE
+				to_chat(user, SPAN_NOTICE("You secure the safety bolts."))
```
Comment (genessee596) [inline code/modules/projectiles/guns/energy/shrapnel.dm:53]:
```suggestion
				to_chat(user, SPAN_NOTICE("You secure the safety bolts, preventing the weapon from destroying empty cells for use as ammuniton."))
```

File: code/modules/projectiles/guns/energy/shrapnel.dm [hunk 2]
```diff
@@ -11,33 +11,49 @@
...
+			else
+				consume_cell = TRUE
+				to_chat(user, SPAN_NOTICE("You loosen the safety bolts."))
```
Comment (genessee596) [inline code/modules/projectiles/guns/energy/shrapnel.dm:56]:
```suggestion
				to_chat(user, SPAN_NOTICE("You loosen the safety bolts, allowing the weapon to destroy empty cells for use as ammunition."))
```

**Review by genessee596 — APPROVED** [review]
Good, but let's make the descriptions more descriptive

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**